Streetcode/Streetcode.BLL/DTO/AdditionalContent/Subtitles/SubtitleDTO.cs

```diff
@@ -8,8 +8,8 @@ public class SubtitleDTO
     public SubtitleStatusDTO SubtitleStatus { get; set; }
     public string FirstName { get; set; }
     public string LastName { get; set; }
-    public string Description { get; set; }
-    public UrlDTO Url { get; set; }
+    public string? Description { get; set; }
```

Comment:
Can we setup Http request/response instead of changing properties to nullable? 
Because  string already  support null value

Reply:
It should be nullable, as in Entities for mapping otherwise it will not be mapped